changelogs/fragments/role_fixes.yml

```diff
@@ -0,0 +1,3 @@
+bugfixes:
+  - roles, the ``static`` property is now correclty set, this will fix issues with ``public`` and ``DEFAULT_PRIVATE_ROLE_VARS`` controls on exporting vars.
```

Comment:
```suggestion
  - roles, the ``static`` property is now correctly set, this will fix issues with ``public`` and ``DEFAULT_PRIVATE_ROLE_VARS`` controls on exporting vars.
```